sourcegraph/about
# Update SG Overview pdf

Closes #5567

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure PDF is updated and loads correctly with the title "Sourcegraph Overview" in the tab
